controllers/user.server.controller.js

```diff
@@ -6,8 +6,21 @@ const User = require('../models/user.server.model');
  * @param res HTTP request response object
  */
 exports.userCreate = function(req, res) {
-    // TODO: implement userCreate()
-    res.json({ dummyTest: "userCreate() dummy test passes" });
+    const username = req.body.username;
+    const name = req.body.name;
+    const email = req.body.email;
+    const hashedPassword = req.body.hashedPassword;
```

Comment:
The password should be hashed in this function, not passed to it. Salted would be good too

Reply:
I've just made a placeholder method to hash the password for now (it just returns a string 'password') and I'll make the hashing of the password into another issue.